homeassistant/strings.json

```diff
@@ -25,6 +25,7 @@
         "confirm_setup": "Do you want to start set up?"
       },
       "data": {
+        "name": "[%key:common::config_flow::data::name%]",
```

Comment:
You sure of this ? lol
Looks like a circular reference !

Reply:
Good catch :laughing: this happens when you do not exclude the source file. Will fix it asap